team_picker/views/index.ejs

```diff
@@ -0,0 +1,16 @@
+<%- include('./partials/header') %>
+
+<h1>List of all the Cohorts</h1>
+
+<% for (let cohort of cohorts) { %>
+  <div class="card" style="width: 18rem;">
+    <img class="card-img-top logo ml-auto mr-auto" src="<%=cohort.logoUrl%>" alt="Card image cap">
+    <div class="card-body">
+      <h5 class="card-title"><%=cohort.name%></h5>
```

Comment:
What happened to `h2`, `h3`, and `h4`? You don't want to jump from `h1` to `h5`. If it's the styling that you're after, you should be using CSS for it (most likely this _shouldn't_ be an `h#` tag at all, as `h#` carries too much semantic meaning that does not apply here).